concourse/scripts/pipecleaner_test.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 var _ = Describe("PipeCleaner", func() {
-	const runTimeout = 2 * time.Second
+	const runTimeout = 5 * time.Second
```

Comment:
Could you split this change out to a separate commit that explains why it was necessary?

Reply:
Hrm, the build now goes green without that change. I've removed it.